packages/formation/sass/modules/_m-back-to-top.scss

```diff
@@ -0,0 +1,73 @@
+//=============================
+// Up To Top Button Styles
+//=============================
+#top-button-container {
+  bottom: 0;
+  pointer-events: none;
+  position: fixed;
+  width: 100%;
+
+  &.va-top-button-container-relative {
+    position: relative;
+    button {
+      transform: translateY(-6px);
+    }
+  }
+
+  .usa-content {
+    position: relative;
+  }
+  &:not(.va-top-button-container-relative) {
+    button.va-top-button-transition-in {
+      transform: translateY(-12px);
+    }
+  }
+
+  button {
+    bottom: 0;
+    background-color: $color-gray-dark;
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+    height: 44px;
+    margin: 0;
+    opacity: 0;
+    padding: 0;
+    pointer-events: none;
+    position: absolute;
+    transition: all 250ms ease-in;
+    right: 0;
+    width: 44px;
+
+    &::after {
+      @include media($medium-screen) {
+        content: 'Back to top';
+      }
+    }
+
+    &::before {
+      font-family: 'Font Awesome 5 Free';
+      font-weight: 900;
+      content: '\F062';
+      margin-right: 0;
+      font-size: 2rem;
+
+      @include media($medium-screen) {
+        margin-right: 0.5em;
+        font-size: 1.5rem;
+      }
+    }
+
+    &:hover {
+      background-color: $color-black;
+    }
+    &.va-top-button-transition-in {
+      pointer-events: all;
+      opacity: 1;
+    }
+    @include media($medium-screen) {
+      padding-left: 17px;
+      padding-right: 19px;
+      width: auto;
+    }
```

Comment:
Prolly should override border-radius here for the right corners so it's not pointy on that end and round on the other.

Reply:
This is how it is supposed to be according to the documentation. From [zeplin](https://app.zeplin.io/project/5e29e7ff922d4d961b5bff4d/screen/5fdbda435599ca472d759b4c)

![image](https://user-images.githubusercontent.com/2008881/103799417-e9a39a00-4fff-11eb-8ede-bb474961c591.png)

Reply:
Ohhhhh, I seeeeeeee.

Is this what you intended, @CrystabelReiter?
![image](https://user-images.githubusercontent.com/12970166/103800138-de04a300-5000-11eb-886c-878b0a1ca514.png)

It feels to me like either it should be hard against the scrollbar on the right—like it is in mobile—or a floating, rounded button.

Reply:
It looks a little funky here, but in Ryan's mocks it's supposed to sit flush on the left of the content 
<img width="428" alt="Screen Shot 2021-01-06 at 3 13 39 PM" src="https://user-images.githubusercontent.com/18386248/103815771-dc070800-5031-11eb-839c-07695d804a31.png">

Reply:
@bkjohnson There still looks like there's a lot of space between the arrow icon and the button label. Can we reduce to 8px between the label and the icon?

Reply:
@cvalarida where did you get that screenshot? Right now the `0.5em` right margin evaluates to 7.5px in storybook, I'll see if I can get it to work with the 8px unit base we have set for va.gov.

![image](https://user-images.githubusercontent.com/2008881/103832715-d14d7280-5033-11eb-862d-4709d6c35e71.png)

Reply:
Oh, I answered this in Slack, but for posterity: I took a screenshot of your gif. :grin: